bigquery/google/cloud/bigquery/job.py

```diff
@@ -1127,11 +1198,44 @@ def from_api_repr(cls, resource, client):
         job._set_properties(resource)
         return job
 
-    def results(self):
+    def query_results(self):
         """Construct a QueryResults instance, bound to this job.
 
         :rtype: :class:`~google.cloud.bigquery.query.QueryResults`
         :returns: results instance
         """
         from google.cloud.bigquery.query import QueryResults
         return QueryResults.from_query_job(self)
+
+    def results(self):
+        """DEPRECATED.
```

Comment:
Can you add a GitHub Issue and a Google Calendar event marking "when" it should be deprecated?